### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 0xchabbi**
fix dead-link and change naming to have right understanding. (Add Peramlink instead of copy path)

### CODE REVIEW

**Comment by 0xchabbi** [comment]
@kmoe @crw

File: docs/README.md [hunk 1]
```diff
@@ -37,5 +37,4 @@ to [the main Terraform CLI documentation](https://www.terraform.io/docs/cli/inde
 
 ## Contribution Guides
 
-* [Maintainer Etiquette](./maintainer-etiquette.md): guidelines and expectations
-  for those who serve as Pull Request reviewers, issue triagers, etc.
+* [Contributing to Terraform](https://github.com/hashicorp/terraform/blob/7ae272fe3184b3c0c6dc066b2d1bc86ac14606bd/.github/CONTRIBUTING.md): a complete guideline for those who want to contribute to this project.
```
Comment (crw) [inline docs/README.md:40]: The correct link is: `.github/CONTRIBUTING.md` as in:

```
[Contributing to Terraform](.github/CONTRIBUTING.md)

**Review by crw — CHANGES_REQUESTED** [review]

**Comment by crw** [comment]
Hi @0xchabbi, thanks for this! Please make the correction indicated and I'd be happy to merge this.

**Comment by 0xchabbi** [comment]
Thank you for the correction! @crw !

**Review by crw — COMMENTED** [review]
lgtm

**Comment by 0xchabbi** [comment]
Hi @crw !
Still showing the 404 page if you click the link. Could we set it with permalink instead of path?

**Comment by crw** [comment]
I see, I was not paying attention to the location of the file. We do not want to have to update this document every time we update `CONTRIBUTING.md`, so hard-linking to a particular snapshot of the file is not desirable. If you can hard-link it so that it always shows the `main` branch version of `CONTRIBUTING.md`, I suppose that would be OK. Alternatively, we can see if you can just use a `../` in the path to refer to the correct relative path, if that is how GitHub thinks about files.

**Comment by crw** [comment]
Yes, that (`../`) fixed it. Thanks for the heads-up!

**Comment by 0xchabbi** [comment]
Good to know how GitHub thinks about files! :) 
Thank you

**Pull request merged**